src/Dialogs/Device/DeviceEditWidget.cpp

```diff
@@ -68,6 +68,7 @@ static constexpr struct {
 #ifdef ANDROID
   { DeviceConfig::PortType::RFCOMM_SERVER, N_("Bluetooth server") },
   { DeviceConfig::PortType::DROIDSOAR_V2, _T("DroidSoar V2") },
+  { DeviceConfig::PortType::GLIDER_LINK, _T("GliderLink receiver") },
```

Comment:
Is it called "GliderLink receiver" or "GliderLink traffic receiver"? (see above)